app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    storage_cache = {}
+
+    def wrapper(*args: int) -> int:
```

Comment:
The type hint `*args: int` is incorrect. `*args` can accept any number of arguments of any type, not just integers. You should use `*args: Any` or omit the type hint for `*args` to allow for more flexibility.